3-networks/README.md

```diff
@@ -54,6 +54,47 @@ If you are not able to use dedicated interconnect, you can also use an HA VPN to
 1. Merge changes to nonprod with `git checkout -b nonprod` and `git push origin nonprod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
 
+### Setup to run via Jenkins
+1. Clone the repo you created manually in bootstrap: `git clone <YOUR_NEW_REPO-3-networks>`
+1. Navigate into the repo `cd YOUR_NEW_REPO_CLONE-3-networks` and change to a non prod branch `git checkout -b plan` (the branch `plan` is not a special one. Any branch which name is different from `dev`, `nonprod` or `prod` will trigger a terraform plan).
+1. Copy contents of foundation to new repo `cp -R ../terraform-example-foundation/3-networks/* .` (modify accordingly based on your current directory).
+1. Copy the `Jenkinsfile` to the root of your new repository (modify accordingly based on your current directory):
+   ```
+   cp ../terraform-example-foundation/build/Jenkinsfile .
+   ```
+1. Update the variables located in the `environment {}` section of the `Jenkinsfile` with values from your environment:
+    ```
+    _POLICY_REPO (optional)
+    _TF_SA_EMAIL
+    _STATE_BUCKET_NAME
+    ```
+1. Copy the `tf-wrapper.sh` configuration file to the root of your new repository (modify accordingly based on your current directory):
+    ```
+    cp ../terraform-example-foundation/build/tf-wrapper.sh .
```

Comment:
I think these instructions cloud use the same format/style used in the cloud build instructions for consistency